docs/squeezing_avr.md

```diff
@@ -2,7 +2,7 @@
 
 AVR is severely resource-constrained, and as QMK continues to grow, it is approaching a point where support for AVR may need to be moved to legacy status as newer development is unable to fit into those constraints.
 
-However, if you need to reduce the compiled size of your firmware, there are a number of options to do so.
+However, if you need to reduce the compiled size of your firmware to fit AVR's limited flash size, there are a number of options to do so.
```

Comment:
I'd opt for leaving out this change.  There are a few smaller ARM controllers that a lot of this would still apply to, and that's not even considering SN32 stuff. Namely STM32's F0 line.

Reply:
Improved this phrase because someone _still_ asked explicitly about impacting flash capacity in Discord after reading this page. The sentence has been updated to with controller agnostic reference.